neural_compressor/torch/algorithms/weight_only/utility.py

```diff
@@ -0,0 +1,443 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+
+from neural_compressor.torch.utils import logger
+
+NF4 = [
```

Comment:
Suggest creating `utils` folder to save utility and constants.
```
└── utils
    ├── constants.py
    ├── utility.py
    └── __init__.py
--

usage:
from xx.weight_only.utils import NF4
```

Reply:
For RTN, utility is enough, You can create the utils when adding AWQ if you prefer.
Remember the utility here is pure for `weight_only`